cmd/galaxy_install.go

```diff
@@ -46,14 +51,39 @@ func (c *GalaxyInstallCommand) Run(args []string) int {
 	}
 
 	galaxyInstall := execCommand("ansible-galaxy", "install", "-r", files[0])
-	logCmd(galaxyInstall, c.UI, true)
+	_, stderr := logCmd(galaxyInstall, c.UI, true)
```

Comment:
Would it be easier if we use [`MockUi`](https://github.com/mitchellh/cli/blob/3d22a244be8aa6fb16ac24af0e195c08b7d973aa/ui_mock.go#L12) here?

Display a spinner instead of the actual `ansible-galaxy` output

Reply:
I thought about something like that... let me see.

Although I thought it might be weird to show the real output for normal install but then not for the "auto update".

Reply:
I don't like all the extra complexity here to capture stdout/err but I'd rather avoid trying to redo galaxy's output.

I'm hoping there's just a simpler way to do what I have here.

Reply:
We can print the actual output after `ansible-galaxy install`, just not in real time

```
mockui.OutputWriter.String()
mockui.ErrorWriter.String()
```

Reply:
How about a9a1ac8 ?

That's the simplest I can come up with. Problem with using plain `UiWriter` is it only uses `Info` so no error info are captured. Which means nothing in `Stderr` either so this is my compromise.

Reply:
And I think a nice side benefit of it is users won't see the now useless blue `WARNING` message.

```
Running command => ansible-galaxy install -r galaxy.yml
- composer (1.7.6) is already installed, skipping.
- ntp (1.6.4) is already installed, skipping.
- logrotate (v0.0.5) is already installed, skipping.
- swapfile (v2.0.22) is already installed, skipping.
- mailhog (2.1.4) is already installed, skipping.

Updating roles: newrelix

Running command => ansible-galaxy install -f -r galaxy.yml newrelix
- changing role newrelix from 0.4.0 to 0.5.0
- downloading role 'trellis-newrelic-php', owned by TypistTech
- downloading role from https://github.com/TypistTech/trellis-newrelic-php/archive/0.5.0.tar.gz
- extracting newrelix to /Users/scott/dev/yes-trellis/vendor/roles/newrelix
- newrelix (0.5.0) was installed successfully
```